File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/stored/library/sync/LookupSyncDirectory.java
package com.lasthopesoftware.bluewater.client.stored.library.sync;

import com.lasthopesoftware.bluewater.client.library.repository.LibraryId;
import com.namehillsoftware.handoff.promises.Promise;

import java.io.File;

public interface LookupSyncDirectory {
	Promise<File> promiseSyncDirectory(LibraryId libraryId);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/stored/library/sync/LookupSyncDirectory.kt
package com.lasthopesoftware.bluewater.client.stored.library.sync

import com.lasthopesoftware.bluewater.client.browsing.library.repository.LibraryId
import com.namehillsoftware.handoff.promises.Promise
import java.io.File

interface LookupSyncDirectory {
	fun promiseSyncDirectory(libraryId: LibraryId): Promise<File>
}
